projects/1.18/assets/bundled-delight/bundledelight/lang/zh_cn.json

```
@@ -0,0 +1,99 @@
+{
+    "block.bundledelight.adzuki_bean_crate": "箱装红豆",
+    "block.bundledelight.adzuki_powder_basket": "箱装红豆粉",
+    "block.bundledelight.black_tea_leaf_crate": "箱装红茶",

```

```suggestion
    "block.bundledelight.black_tea_leaf_crate": "箱装红茶叶",
```
Farmer's Respite/农夫暇事兼容存储方块，名称遵照库内农夫暇事译名